core/src/main/scala/com/typesafe/tools/mima/core/MemberInfo.scala

```diff
@@ -7,8 +7,9 @@ object MemberInfo {
  /** The index of the string $_setter_$ in this string */
   private def setterIdx(name: String) = name.indexOf(setterTag)
 
-  private val setterTag = "$_setter_$"
+  private val setterTag = "$_setter_"
```

Comment:
Can't remember the exact failing test now but there was something that didn't match with it. Maybe it's by now just a remnant of an earlier attempt. I'll try to back it out later, to minimize the changes.